src/types/datasets.tsx

```diff
@@ -7,25 +7,25 @@ export type LayerTypes = Readonly<{
   gs_name: string;
   gs_dimension: string;
   range: string;
-  author: null;
+  author: string;
```

Comment:
```suggestion
  author: string | null;
```
I think you are doing a trick. The author from the API could be null. But you are lying the TS with the placeholder.